content/blog/en/dojo-cli-template-app.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Dojo CLI Template App
```

Comment:
Let's give this more of an english title. Something like "Scaffolding your first dojo app with the dojo cli" or something like that?